.remarkrc

```diff
@@ -1,5 +1,2 @@
 {
-  "plugins": [
-    "@1stg/preset"
```

Comment:
@JounQin This cause `lint` script fails, maybe dependencies is outdated? Would you mind take a look? https://github.com/prettier/eslint-plugin-prettier/actions/runs/13671651574/job/38222988172#step:7:15